Makefile

```diff
@@ -32,10 +32,10 @@ generate: ## Generate code and manifests
 format: ## Format the source code
 	$(Q)go fmt ./...
 
-lint: ## Run golangci-lin
-	$(Q)go run github.com/golangci/golangci-lint/cmd/golangci-lint run
+lint: ## Run golangci-lint
+	$(Q)go run github.com/golangci/golangci-lint/cmd/golangci-lint run --fast
```

Comment:
See my above comment: I think we should be using a custom .golangci.yml to drive configuration vs. supplying flags when performing the linting process.